versioned/persist/mongodb/src/main/java/org/projectnessie/versioned/persist/mongodb/MongoDatabaseAdapter.java

```diff
@@ -96,18 +103,27 @@ public void eraseRepo() {
     client.getCommitLog().deleteMany(idPrefixFilter);
     client.getKeyLists().deleteMany(idPrefixFilter);
     client.getRefLog().deleteMany(idPrefixFilter);
+    client.getBlobs().deleteMany(idPrefixFilter);
   }
 
   private Document toId(Hash id) {
+    return toId(id.asString());
+  }
+
+  private Document toId(String id) {
     Document idDoc = new Document();
     // Note: the order of `put` calls matters
     idDoc.put(ID_REPO_NAME, repositoryId);
-    idDoc.put(ID_HASH_NAME, id.asString());
+    idDoc.put(ID_HASH_NAME, id);
     return idDoc;
   }
 
-  private List<Document> toIds(Collection<Hash> ids) {
-    return ids.stream().map(this::toId).collect(Collectors.toList());
+  private List<Document> toIdsStream(Stream<String> ids) {
```

Comment:
nit: do we have to rename this method? It looks like there are no overloads :thinking: